synapse/handlers/groups_local.py

```diff
@@ -303,7 +303,7 @@ async def create_group(self, group_id, user_id, content):
             content["user_profile"] = await self.profile_handler.get_profile(user_id)
 
             try:
-                res = await self.transport_client.create_group(
+                res = await self.groups_server_handler.create_group(
```

Comment:
I'm unsure about this change, the function signature and such doesn't match. I suggest we don't do this, but file an issue with our findings and solve it outside of this PR.